css-text-decor-4/Overview.bs

```diff
@@ -339,15 +339,17 @@ Text Decoration Line Thickness: the 'text-decoration-thickness' property</h3>
 		<dt><dfn><<length>></dfn>
 		<dd>
 			Specifies the thickness of text decoration lines as a fixed length.
-			The UA must floor the actual value at one device pixel.
+			The UA should round the actual value to the nearest integer,
```

Comment:
WDYT of this rephrasing?
`The UA should round to the nearest positive integer.`

This removes the mention of "device pixel" which is difficult for implementations, and is a little simpler.

Reply:
Sounds good to me.